Website/Portals/_default/HotcakesViews/Porto5/Views/Checkout/PaymentError.cshtml

```diff
@@ -41,7 +41,7 @@
                     <div class="row">
                         <div class="col-12 col-md-8 offset-md-2 text-center mt-4 mb-5">
                             <a class="btn btn-secondary" href="@Url.RouteHccUrl(HccRoute.Checkout, new { action = "cancel" })">@Localization.GetString("CancelOrder")</a>
-                            <input type="submit" id="btnsubmit" class="btn btn-primary" value="@Localization.GetString("PlaceOrder")" />
+                            <input type="submit" id="hcTakeOrder" class="btn btn-primary" value="@Localization.GetString("PlaceOrder")" />
```

Comment:
@VAlmea Is this a new update, or are you restoring it back to the hcTakeOrder class? What was the original value?

Reply:
I changed the btnsubmit Id to hcTakeOrder in the PaymentError view in order to reuse all the logic from the checkout view. I reviewed all the js files and the btnsubmit Id for the button is not used.